viewing_party/main.py

```diff
@@ -0,0 +1,237 @@
+
+def create_movie(title, genre, rating):
+    if title == None:
+        return None
+    elif genre == None:
+        return None
+    elif rating == None: 
+        return None
```

Comment:
This is totally fine, but could be a little more succinct by using `or` to but all this on one line, since the result is the same in any case.

Reply:
if title = None or genre == None etc...